src/components/InventoryManager/EditListingPopup.js

```diff
@@ -300,7 +364,7 @@ export default function EditListingPopup({
     }
   );
   return (
```

Comment:
Are all of the TODO comments still necessary? (Line 190, 97, etc + in some of your other files)

Reply:
oh actually these are all taken out in the other pull request but i will fix here too !!